canister/src/api/get_utxos.rs

```diff
@@ -145,6 +145,19 @@ fn get_utxos_internal(
     }
 }
 
+fn max_confirmations_count(
```

Comment:
* I'd replace the term confirmations here with "depth". That's the more accurate term, and that is also the term used in the wiki.
* I think it'd be a better abstraction if we have a method called `get_stability_count` that encapsulates everything that's needed to compute the stability count, rather than having part of the logic in this method and the other part in `get_utxos_from_chain`.

Reply:
That is a good idea, I will do that.

Reply:
Are you sure that we should use "depth" everywhere, instead of "confirmation"? Because by the article in [wiki](https://wiki.internetcomputer.org/wiki/Bitcoin_integration), the depth of the tree with only the root node is 1, which is opposite to the general definition of the tree depth (in this case 0). Do you think it may be confusing to the reader? What do you think about using "confirmation" in code as well as in the wiki instead of "depth"?